next/src/lib/orm/data-source.ts

```diff
@@ -1,15 +1,19 @@
-import * as dotenv from 'dotenv';
-import * as path from 'path';
+// import * as dotenv from 'dotenv';
+// import * as path from 'path';
 import { DataSource, ObjectLiteral, ObjectType, Repository } from 'typeorm';
 
 import { DataCheck } from './entity/DataCheck';
+import { Dataset } from './entity/Dataset';
+import { DatasetAccess } from './entity/DatasetAccess';
+import { Organization } from './entity/Organization';
+import { OrganizationMembership } from './entity/OrganizationMembership';
 import { Record } from './entity/Record';
 import { User } from './entity/User';
 
 // Don't remove next lines
-dotenv.config({
-  path: path.resolve(__dirname, '../../../.env.local'),
-});
+// dotenv.config({
+//   path: path.resolve(__dirname, '../../../.env.local'),
+// });
```

Comment:
```suggestion
```

Is it ok to remove this completely?

Reply:
I would prefer not to remove this code as I use it when creating/applying migrations. I prefer running next.js on my machine rather than in a Docker container. This is slightly faster for development.